### ISSUE TITLE
Export with null/undefined polygons not quite working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#145]
```
In GeoDa 1.5.35

1) Open a shapefile with some undefined polygons
2) Notice that when selecting all objects on a map, some objects in the table 
remain undefined.  Also, notice that when selecting Save Mean Centers from a 
Map, some entries are blank in the table, not (0,0).
3) Export to GeoJSON.
4) Notice that the previously undefined objects are tiny points/polygons at 
(0,0).  This can be verified by saving the mean centers again.


```

Original issue reported on code.google.com by `mmcc...@gmail.com` on 7 May 2014 at 8:52

**Comment by lixun910** [comment]
can't find a data. Besides, since the data reading is switching to OGR, the null/undefined should be handled by OGR drivers.

**Comment by jkoschinsky** [comment]
Here are data with undefined/empty cells. When you export the file as a new shapefile, these cells are converted to zeros. Is there a way to keep these undefined/empty with OGR after exporting?

[Data.zip](https://github.com/lixun910/geoda/files/112914/Data.zip)

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.87 (OSX Yosemite, 10.10.5 and Win 7 Professional (64-bit)).

numeric null fields are automatically converted to zero
